modules/troubleshooting-network-observability-after-installation.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * networking/network_observability/troubleshooting-network-observability.adoc
+
+:_content-type: PROCEDURE
+[id="configure-network-traffic-console_{context}"]
+= Configuring network traffic menu entry in the {prouct-title} console
+
+Manually configure the network traffic menu entry in the {product-title} console where network traffic menu entry is not connected to the network.
+
+.Prerequisites
+
+* You have installed {product-title} version 4.10+.
+
+.Procedure
+
+. Check if the `spec.consolePlugin.register` field is set to `true` by running the following command:
++
+[source,terminal]
+----
+$ oc -n netobserv get flowcollector cluster -o yaml
+----
++
+.Example output
+----
+apiVersion: flows.netobserv.io/v1alpha1
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  consolePlugin:
+    register: true
+----
+
+. Optional: Set the `spec.consolePlugin.register` field to `true` by running the following command:
++
+[source,terminal]
+----
+$ oc -n netobserv edit flowcollector cluster -o yaml
+----
++
+.Example output
+----
+apiVersion: flows.netobserv.io/v1alpha1
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  consolePlugin:
+    register: true
+----
+
```

Comment:
We should add the following mention just in case (coming from [upstream FAQ](https://github.com/netobserv/network-observability-operator/blob/main/FAQ.md#there-is-no-network-traffic-menu-entry-in-openshift-console)):

If for any reason the registration seems to have failed, you can still do it manually by editing the Console Operator config:

```bash
kubectl edit console.operator.openshift.io cluster
```

If it's not already there, add the plugin reference:

```yaml
spec:
  plugins:
  - netobserv-plugin
```

Reply:
And the following one [I've just added](https://github.com/netobserv/network-observability-operator/pull/234):

Ensure console pods are all in `Running` state using the following command:

```bash
oc get pods -n openshift-console -l app=console
```

If you had previously used the console with the plugin installed, you may need to restart console pods to clear cache:

```bash
oc delete pods -n openshift-console -l app=console
```

Reply:
Addressed